jetty-ee10/jetty-ee10-tests/jetty-ee10-test-client-transports/src/test/java/org/eclipse/jetty/ee10/test/client/transport/ServerTimeoutsTest.java

```diff
@@ -597,15 +597,16 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
         BlockingQueue<Object> objects = new LinkedBlockingQueue<>();
         CountDownLatch clientLatch = new CountDownLatch(1);
         client.newRequest(newURI(transport))
-            .onResponseContentAsync((response, content, callback) ->
+            .onResponseContentAsync((response, chunk, demand) ->
             {
-                objects.offer(content.remaining());
-                objects.offer(callback);
+                objects.offer(chunk.remaining());
+                chunk.release();
+                objects.offer(demand);
             })
             .send(result ->
             {
                 objects.offer(-1);
-                objects.offer(Callback.NOOP);
+                objects.offer((Runnable)() -> {});
```

Comment:
Maybe just offer `null`, and add a null-check below?

Reply:
For a test? I prefer to avoid null checks.